cartridge-cli.lua

```diff
@@ -1728,19 +1732,23 @@ end
 
 local function construct_build_image_dockerfile()
     -- The application build dockerfile consent theese parts:
-    -- - dockerfile_base_layers: the base image (passed in app_state.dockerfile_base_layers)
+    -- - build_base_dockerfile_layers: the base image
     -- - prepare: installing packages required for build (git gcc make cmake unzip)
     --            and creating tarantool user and directories
     -- - install_tarantool: install Tarantool on image
 
+    if app_state.build_base_dockerfile_layers == nil then
```

Comment:
Should we use some default value in this case?

Reply:
The default value for base dockerfiles is set in `cmd_pack.parse` funcion. 
Here we have just `app_state` with all values set.

Reply:
Hm... Is this check redundant then?
Or it should be an assert (because can happen only in dev)?

Reply:
Yes, I agree that it should be an assert.

Reply:
BTW, [here](https://github.com/tarantool/cartridge-cli/pull/148) asserts are caught and logged correclty.